app/models/reservation.rb

```diff
@@ -6,8 +6,12 @@ class Reservation < ApplicationRecord
   delegate :name, to: :room, prefix: true
 
   def unique_room_date
-    return unless Reservation.where(room_id:, date:).exists?
+    # return unless Reservation.where(room_id:, date:).exists?
 
-    errors.add(:base, 'This room is already reserved for this date')
+    # errors.add(:base, 'This room is already reserved for this date')
+  end
```

Comment:
Great Job so far 👏 Kindly remove comments from your code if you no longer need to use it or uncomment this code if you still plan on limiting the dates of the reservation. This will help your code look more professional and work as expected 👍 .

Reply:
Hello, @sebalp100. When I uncomment the code my linters do not pass.

Reply:
@enoqueJonas , you can try using this code:

```
def unique_room_date
    return unless Reservation.where(room_id:, date:).exists?
    errors.add(:base, 'This room is already reserved for this date')
end

```